### ISSUE TITLE
tensorflow-serving with custom-op, when start tensorflow_model_server, got Op type not registered 'ActiveShift2DOp' in binary running on server.

### ISSUE THREAD

**Issue by Eunhui-Kim** [tensorflow/custom-op#68]
Hello

My Tensorflow ver 1.15.3.

I built custom-op as the guide.

I made shift_op custom-op and make it package as wheel and installed it.

by make package in the makefile
 active_shift_pip_pkg: $(ACTIVE_SHIFT_TARGET_LIB)
    ./build_pip_pkg.sh make artifacts`

using the wheel file under artifacts directory, 
I can use the custom op by calling.

`from tensorflow_active_shift.python.ops import active_shift2d_ops`

After I built the model using tensorpack with my new custom-op, shift and I got checkpoint, and meta file.
and by making the generation .py file as follows I can get pb. file

` 
   import tensorflow as tf
   from tensorflow_active_shift.python.ops import active_shift2d_ops
   from tensorflow.python.saved_model import builder as saved_model_builder
   from tensorflow.python.saved_model import signature_constants, signature_def_utils, tag_constants, utils
   from tensorflow.python.util import compat
   import os
 
  tf.contrib.resampler
 
  config = tf.ConfigProto(allow_soft_placement=True)
  sess = tf.Session(config=config)
 
  saver = tf.train.import_meta_graph('./train_log/cifar10-Shift20/graph-0629-155507.meta')
  saver.restore(sess, tf.train.latest_checkpoint('./train_log/cifar10-Shift20/'))
 
  graph = tf.get_default_graph()
  model_signature = tf.saved_model.signature_def_utils.predict_signature_def(
     inputs={'image':graph.get_tensor_by_name('input:0')},
     outputs={'label':graph.get_tensor_by_name('label:0')})
 #print(inputs, outputs)

 model_path = "./model"
 model_version = 1
 export_path = os.path.join(compat.as_bytes(model_path), compat.as_bytes(str(model_version)))
 builder = saved_model_builder.SavedModelBuilder(export_path)
 builder.add_meta_graph_and_variables(
     sess, [tag_constants.SERVING],
     clear_devices=True,
     signature_def_map={
         signature_constants.DEFAULT_SERVING_SIGNATURE_DEF_KEY: model_signature,
     })
 builder.save()`

I run 
`tensorflow_model_server --port=8501 --rest_api_port=8502 --model_base_path="/path /model/"`

but I got the error as follows.

SavedModel load for tags { serve }; Status: fail: Not found: Op type not registered 'ActiveShift2DOp' in binary running on grace-Alienware-Area-51-R5. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.. 

How can I solve this issue?

Since version problem, I can't add tf.contrib.resampler in the tensorflow, tensorpack code.
When I just use tf.disable_eager_execution(), I can check the InferenceGraph also include the new op as follows. 

`for op in graph.get_operations():
    print(op.name)`
...
InferenceTower/res3.1/shift1/ActiveShift2DOp/ActiveShift2DOp/shift
....

How can this problem solve?

Thank you in advance~

**Comment by Eunhui-Kim** [comment]
It was not custom-op problem.
I solved the problem by changing wrapping layer.
Thank you
